cli/index.js

```diff
@@ -309,7 +309,13 @@ export async function main(argv, options) {
   assemblyscript.setSharedMemory(compilerOptions, opts.sharedMemory);
   assemblyscript.setImportTable(compilerOptions, opts.importTable);
   assemblyscript.setExportTable(compilerOptions, opts.exportTable);
-  assemblyscript.setExportStart(compilerOptions, typeof opts.exportStart === "string" ? opts.exportStart : null);
+  if (opts.exportStart) {
+    if (typeof opts.exportStart === "string" && opts.exportStart !== "") {
+      assemblyscript.setExportStart(compilerOptions, opts.exportStart);
+    } else {
+      return prepareResult(Error("exportStart option required string argument"));
+    }
+  }
```

Comment:
An alternative would be to assume `_start` as the default if just `--exportStart` is given, effectively making the name optional. Would that preferable over an error, also given that it then behaves like `--explicitStart` did before, just with a different name?

Reply:
okey!